pkg/loadbalancer/experimental/script_test.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package experimental
+
+import (
+	"bufio"
+	"context"
+	"maps"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/cilium/hive/cell"
+	"github.com/cilium/hive/hivetest"
+	"github.com/cilium/hive/script"
+	"github.com/cilium/hive/script/scripttest"
+	"github.com/cilium/statedb"
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	daemonk8s "github.com/cilium/cilium/daemon/k8s"
+	"github.com/cilium/cilium/pkg/datapath/tables"
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/k8s/client"
+	"github.com/cilium/cilium/pkg/k8s/testutils"
+	"github.com/cilium/cilium/pkg/k8s/version"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+func TestScript(t *testing.T) {
+	version.Force(testutils.DefaultVersion)
```

Comment:
I had an earlier version that allowed doing `--k8s-force-version=1.23` in the `txtar` files, but it really wrecks havoc with the `version` package globals, so ended up removing the test I had for older k8s versions (e.g. Endpoints no EndpointSlices). For now I'm just forcing the version here. Will have to revisit the `pkg/k8s/version` global variable issue in a later PR instead.

Reply:
Let's create an issue and link it here maybe

Reply:
https://github.com/cilium/cilium/issues/35537